documentation/asciidoc/accessories/camera/v4l2.adoc

```diff
@@ -1,58 +1,44 @@
-== The V4L2 Driver
+== V4L2 Drivers
 
-The V4L2 driver provides a standard Linux driver for accessing camera features: this is the driver needed to use a Raspberry Pi camera as, for example, a webcam. The V4L2 driver provides a standard API on top of the firmware-based camera system.
+V4L2 drivers provide a standard Linux interface for accessing camera and codec features. They are loaded automatically when the system is started, though in some non-standard situations you may need to xref:camera.adoc#if-you-do-need-to-alter-the-configuration[load camera drivers explicitly].
 
-=== Installing the V4L2 Driver
+=== Driver differences when using `libcamera` or the legacy stack
```

Comment:
This implies that V4L2 is usable with either the legacy stack or the libcamera stack, but `csi-2-usage.adoc` says the V4L2 interface is only usable with the libcamera stack?

Reply:
There are several different V4L2 drivers, but it's the Unicam (CSI-2 receiver) one that's only available when using libcamera. I'll clarify that.